src/main/java/org/reaktivity/nukleus/sse/internal/stream/ServerStreamFactory.java

```diff
@@ -512,8 +508,8 @@ private void handleAbort(
         private void setHttpResponseHeaders(
             ListFW.Builder<HttpHeaderFW.Builder, HttpHeaderFW> headers)
         {
-            headers.item(h -> h.name(":status").value("200"));
-            headers.item(h -> h.name("content-type").value("text/event-stream"));
+            headers.item(h -> h.representation((byte)0).name(":status").value("200"));
+            headers.item(h -> h.representation((byte)0).name("content-type").value("text/event-stream"));
```

Comment:
`.representation((byte)0)` no longer required, now defaulted in generated flyweights.